specs/TryUpdateRuntime.md

```diff
@@ -0,0 +1,229 @@
+# Background
+The new version of an app might requires a newer version of Edge WebView2 Runtime. The app updater might want to ensure that the newer version of Edge WebView2 Runtime
+is installed before updating the app to the newer version. The app could also update to newer version with some feature disabled, and then request update so that it
+could move to newer version of Edge WebView2 Runtime and enable those features faster.
+Edge WebView2 Runtime is auto updated and normally the latest version should be already installed. However, there could be cases that we need trigger Edge WebView2 Runtime
+update to ensure coordinated app and runtime update.
+
+# Description
+You may call the `TryUpdateRuntime` API to check and install updates to installed Edge WebView2 Runtime. This is useful when the app wants to coordinate app and
+Edge WebView2 Runtime update.
+
+# Examples
+## .NET, WinRT
+```c#
+using Microsoft.Web.WebView2.Core;
+async protected bool EnsureWebView2RuntimeVersion(string minimalVersionRequired)
+{
+    string currentRuntimeVersion = CoreWebView2Environment.GetAvailableBrowserVersionString();
+    if (CoreWebView2Environment.CompareBrowserVersions(currentRuntimeVersion, minimalVersionRequired) < 0)
+    {
+      auto environment = await CoreWebView2Environment.CreateAsync();
+      auto updateResult = await environment.TryUpdateRuntimeAsync();
+      if (updateResult.UpdateRuntimeStatus != CoreWebView2RuntimeUpdateStatus.Updated)
+        return false;
+    }
+    // check runtime version again
```

Comment:
SAMPLE: This checks twice in the common case that everything was already good. Should probably move the recheck inside the "if" block.